airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -350,6 +361,13 @@ def execute(self, context) -> Optional[str]:
                 "--project",
                 self.project_id,
             ]
+            if self.impersonation_chain:
```

Comment:
Since we only support one element, we should throw in an exception when someone passes several elements in an array.

Reply:
Make sense, Thanks. 👍